nf_core/modules/create.py

```diff
@@ -272,27 +283,46 @@ def render_template(self):
             template_stat = os.stat(os.path.join(os.path.dirname(nf_core.__file__), "module-template", template_fn))
             os.chmod(dest_fn, template_stat.st_mode)
 
-    def get_repo_type(self, directory):
+    def get_repo_type(self):
         """
         Determine whether this is a pipeline repository or a clone of
         nf-core/modules
         """
         # Verify that the pipeline dir exists
-        if dir is None or not os.path.exists(directory):
-            raise UserWarning(f"Could not find directory: {directory}")
-
-        readme = os.path.join(directory, "README.md")
-        # Determine repository type
-        if os.path.exists(readme):
-            with open(readme) as fh:
-                if fh.readline().rstrip().startswith("# ![nf-core/modules]"):
-                    return "modules"
-                else:
-                    return "pipeline"
-        else:
+        if dir is None or not os.path.exists(self.directory):
+            raise UserWarning(f"Could not find directory: {self.directory}")
+
+        # Try to find the root directory
+        base_dir = os.path.abspath(self.directory)
+        config_path_yml = os.path.join(base_dir, ".nf-core.yml")
+        config_path_yaml = os.path.join(base_dir, ".nf-core.yaml")
```

Comment:
It is now, as of this PR when we add the file to the template. I took this logic because it's already done in `nf_core.utils.load_tools_config()`. Originally I added that support due to my experience in MultiQC of `multiqc_config.yml` files. It can be a super annoying bug and quite difficult to spot.

But yeah, we can just strip out the `yaml` support I guess.. Might be simpler 🤷🏻

Reply:
I see your point -- either is fine